system-test/read-rows.ts

```diff
@@ -88,9 +101,9 @@ describe('Bigtable/Table', () => {
       responses = null;
       rowKeysRead = [];
       requestedOptions = [];
-      stub = sinon.stub(bigtable, 'request').callsFake(cfg => {
-        const reqOpts = cfg.reqOpts;
-        const requestOptions = {};
+      stub = (sinon as any).stub(bigtable, 'request').callsFake(cfg => {
```

Comment:
Should install `@types/sinon` for this and use `sinon.SinonStub`

Reply:
I did!  There is some weird typing bug I didn't want to try digging into here, and figured I would pick it up later.